sparse/etc/pulse/xpolicy.conf.d/xvars.conf

```diff
@@ -0,0 +1,2 @@
+[variable]
+droid_source_input_fmradio = input-fm_rx
```

Comment:
Are there new changes related to this?
Also @Nokius we were supposed to add this file back right?
https://github.com/CancroSailors/droid-config-cancro/commit/449f300367e059b6a7a91edbf10248a94a57e135#diff-1e4d37b9ace0f37dc5ad861791b9688fL66 

for the line about route_sources_first .. any idea why it is no longer needed?

Reply:
On Mi 4 everything's fine without that file ( arm_qualcomm_msm_8974_cancro_flattened_device_tree_000b.pa )

Reply:
@alireza1219 oh that file was added for fixing the radio - without it it used to take the audio source from the mic instead of radio, even in the radio app. Not sure if they changed pulseaudio policy lately..